cpp/src/Slice/SliceUtil.cpp

```diff
@@ -605,3 +624,137 @@ Slice::getTypeScopedName(const TypePtr& type)
         return contained->scoped();
     }
 }
+
+string
+Slice::relativePath(const string& path1, const string& path2)
```

Comment:
I moved it from JsUtil, but this end-up being used only slice2js, so I should probably move it back to slice2js.